assets/prefabs/characters/marketCitizen.prefab

```diff
@@ -41,7 +41,7 @@
         ]
     },
     "Behavior" : {
-        "tree" : "Behaviors:stray"
+        "tree" : "MetalRenegades:shopkeeperStray"
```

Comment:
Yep I completely agree to this. Wonder what the fields would be in the restrictedStray behavior though. The building entity in which Actor should reside in?

Reply:
I agree that this behavior would be more useful in a generic form. I made a new generic `restrictedStray` behavior in Behaviors, and used that behavior here. Instead of referencing a building entity, the Behaviors version uses a bounding `Rect2i` from a `StrayRestrictionComponent`.

Here's the pull request: https://github.com/Terasology/Behaviors/pull/41/